docs/DeveloperGuide.adoc

```diff
@@ -742,7 +742,7 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 (For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
 
 [discrete]
-=== Use case: Delete person
+=== Use case: UC1 - Delete person
 
 *MSS*
 
```

Comment:
Should we add guarantee for this use case?
"Contact information of person will be removed from the application if the contact information of the person specified exists"